ste/xferRetrypolicy.go

```diff
@@ -220,6 +223,12 @@ func NewBFSXferRetryPolicyFactory(o XferRetryOptions) pipeline.Factory {
 				switch {
 				case err == nil:
 					action = "NoRetry: successful HTTP request" // no error
+				case response.Response() != nil && response.Response().StatusCode == http.StatusConflict:
+					if o.RetryHTTPErrorConflict && os.Getenv("AZCOPY_DISABLE_CUSTOM_RETRY") == "" {
```

Comment:
Why do we need to check 2 flags here if config is set just retry

Reply:
We want to retry only on destination pipleine and not source.

Reply:
o.RetryHTTPErrorConflict :  This represent the flag which is true as harcoded value for now
os.Getenv("AZCOPY_DISABLE_CUSTOM_RETRY") :  What does this signify ? Appears we check this shall be unset to make this retry happen.

Reply:
If you want to disable retry on 409, you can set this environment var

Reply:
I don't think we need the global variable right now. The custom error code retries were intended to be something we'd address in 10.21, right?

If so, then there's not really a need to implement a flag like this right now, especially since it's not really getting used?

Reply:
The flag is not to support a set of error codes. But if the user has to disable retrying on 409 for some reason, it is easier to toggle env than switch builds